coverage.json

```diff
@@ -0,0 +1 @@
olumn":8},"end":{"line":135,"column":57}},"30":{"start":{"line":139,"column":8},"end":{"line":139,"column":33}},"31":{"start":{"line":143,"column":8},"end":{"line":143,"column":54}},"32":{"start":{"line":147,"column":8},"end":{"line":147,"column":82}},"33":{"start":{"line":148,"column":8},"end":{"line":148,"column":112}},"34":{"start":{"line":151,"column":8},"end":{"line":151,"column":49}},"35":{"start":{"line":155,"column":8},"end":{"line":155,"column":53}},"36":{"start":{"line":156,"column":8},"end":{"line":156,"column":57}},"37":{"start":{"line":160,"column":8},"end":{"line":160,"column":54}},"38":{"start":{"line":161,"column":8},"end":{"line":161,"column":42}},"39":{"start":{"line":162,"column":8},"end":{"line":162,"column":5870}},"40":{"start":{"line":163,"column":12},"end":{"line":163,"column":5936}},"41":{"start":{"line":169,"column":8},"end":{"line":169,"column":150}},"42":{"start":{"line":170,"column":8},"end":{"line":170,"column":43}},"43":{"start":{"line":174,"column":8},"end":{"line":174,"column":67}},"44":{"start":{"line":175,"column":8},"end":{"line":175,"column":81}},"45":{"start":{"line":176,"column":8},"end":{"line":176,"column":57}},"46":{"start":{"line":177,"column":8},"end":{"line":177,"column":58}},"47":{"start":{"line":179,"column":8},"end":{"line":179,"column":50}}},"branchMap":{"1":{"line":35,"type":"if","locations":[{"start":{"line":35,"column":8},"end":{"line":35,"column":8}},{"start":{"line":35,"column":8},"end":{"line":35,"column":8}}]},"2":{"line":40,"type":"if","locations":[{"start":{"line":40,"column":8},"end":{"line":40,"column":8}},{"start":{"line":40,"column":8},"end":{"line":40,"column":8}}]},"3":{"line":45,"type":"if","locations":[{"start":{"line":45,"column":8},"end":{"line":45,"column":8}},{"start":{"line":45,"column":8},"end":{"line":45,"column":8}}]},"4":{"line":50,"type":"if","locations":[{"start":{"line":50,"column":8},"end":{"line":50,"column":8}},{"start":{"line":50,"column":8},"end":{"line":50,"column":8}}]},"5":{"line":51,"type":"if","locations":[{"start":{"line":51,"column":8},"end":{"line":51,"column":8}},{"start":{"line":51,"column":8},"end":{"line":51,"column":8}}]},"6":{"line":56,"type":"if","locations":[{"start":{"line":56,"column":8},"end":{"line":56,"column":8}},{"start":{"line":56,"column":8},"end":{"line":56,"column":8}}]},"7":{"line":61,"type":"if","locations":[{"start":{"line":61,"column":8},"end":{"line":61,"column":8}},{"start":{"line":61,"column":8},"end":{"line":61,"column":8}}]},"8":{"line":66,"type":"if","locations":[{"start":{"line":66,"column":8},"end":{"line":66,"column":8}},{"start":{"line":66,"column":8},"end":{"line":66,"column":8}}]},"9":{"line":107,"type":"if","locations":[{"start":{"line":107,"column":8},"end":{"line":107,"column":8}},{"start":{"line":107,"column":8},"end":{"line":107,"column":8}}]},"10":{"line":163,"type":"if","locations":[{"start":{"line":163,"column":12},"end":{"line":163,"column":12}},{"start":{"line":163,"column":12},"end":{"line":163,"column":12}}]}}}}
```

Comment:
This file contains absolute path like `/Users/khaihkd/projects/tomo/tomo-master/contracts/BlockSigner.sol`
Either this is a generated file that should not be versioned, either it should not contain this path